av_metrics/src/video/mod.rs

```diff
@@ -198,7 +195,6 @@ trait VideoMetric {
         frame2: &FrameInfo<T>,
     ) -> Result<Self::FrameResult, Box<dyn Error>>;
 
-    #[cfg(feature = "decode")]
```

Comment:
Curious what is the use case for this change?

Reply:
My plan is to use bus, crossbeam and rayon to have the cli fast by running frames in parallel, so basically I'll rip apart and redo some of the abstractions to fit within that.